### DISCUSSION TITLE
This is so cooooool!!!where is the source code of this?

### DISCUSSION THREAD

**Discussion by liuszhang** [Blazor-Diagrams/Blazor.Diagrams#473] [Q&A]
I saw this on the official website：https://blazor-diagrams.zhaytam.com/

![diagram](https://github.com/user-attachments/assets/d9ebbc04-6fca-4312-8abf-b8b48902a772)

**Comment by write2varun** [comment]
you can find the page in the repository Blazor.Diagrams-develop\site\Site\Components\Landing\LandingShowcaseDiagram.razor
